### ISSUE TITLE
Debug local object recognition

### ISSUE THREAD

**Issue by jweisz** [tjbot-ce/node-tjbotlib#2]
Local object recognition tends to return a lot of irrelevant, high-confidence items. This needs to be debugged (akin to how face detection was debugged -- we might need to switch to a different model).

**Comment by jweisz** [comment]
It seems there were 2 issues:
1. Object detection was assuming YOLO-style output but we're using SSD MobileNet v2.
2. Image classification was treating logits as probabilities.

Both issues have now been fixed but more robust testing is needed to see if TJBot recognizes actual objects.
